marketplace_docs/templates/Fabric/fabfile.py

```diff
@@ -74,7 +74,7 @@ def install_pkgs():
     print("--------------------------------------------------")
     run("apt-get -o DPkg::Lock::Timeout=120 -qqy update")
     run(
-        'apt-get -o DPkg::Lock::Timeout=120 -qqy -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" upgrade'
+        'DEBIAN_FRONTEND=noninteractive apt-get -o DPkg::Lock::Timeout=120 -qqy -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" upgrade'
```

Comment:
There is `run("DEBIAN_FRONTEND=noninteractive")` several lines above this (shouldn't it be doing `export` to even have a chance to be applied? I'm confused actually why it's the way it is), but it doesn't work  (even with the `export` tweak) unless explicitly included at the beginning of the command.

Reply:
Yeah that doesn't work at all because each `run` does a new fork. I think just setting `DEBIAN_FRONTEND=noninteractive` could work or writing `env DEBIAN_FRONTEND=noninteractive` as the command could work, but this doesn't.

Reply:
So we probably also need this on the other `apt-get` right below this, and we should remove the bare `DEBIAN_FRONTEND=noninteractive` above.

Reply:
Tweaked and tested the build still works